components/ThemeToggleButton.jsx

```diff
@@ -0,0 +1,22 @@
+import { useContext } from "react";
+import styles from "@/styles/components/theme-toggle-button.module.scss";
+import PageFeatures from "@/server/context";
+import ToggleButton from "@/components/ToggleButton";
+
+export default function ThemeToggleButton() {
+	const theme = useContext(PageFeatures)["theme"];
+	const setTheme = useContext(PageFeatures)["setTheme"];
+	const ToggleTheme = () => {
+		setTheme((prev) => {
+			const newValue = prev === "bright" ? "dark" : "bright";
+			localStorage.setItem("theme", newValue);
+			return newValue;
+		});
+	};
+	return (
+		<section className={styles["toggle-theme"]}>
```

Comment:
as I described in another comment, the section, h2 should be in the page itself.
this component is only a wrapper to ToggleButton

Reply:
noted